jib-cli/src/main/java/com/google/cloud/tools/jib/cli/ActualTimestampProvider.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli;
+
+import com.google.cloud.tools.jib.api.AbsoluteUnixPath;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.function.BiFunction;
+
+/** A timetamp provider that uses the actual file last-modification time from the file-system. */
+class ActualTimestampProvider implements BiFunction<Path, AbsoluteUnixPath, Instant> {
```

Comment:
Is this used somewhere? I can't find it. I'm wary of people using this provider if it's going to be used for creating an image, because it greatly undermines reproducibility of a layer.

Reply:
It's pulled in from `LayerParserDefinition#parseTimestampsDirective()`.

Reply:
It's pulled in from `LayerParserDefinition#parseTimestampsDirective()`.

Reply:
It's pulled in from `LayerParserDefinition#parseTimestampsDirective()`.

Reply:
So the question is, should we drop support for `ts=actual`? Jib plugins don't support it (and will never after we build an extension framework). I find think this is very useful, especially when considering the negatives. If someone really wanted to set it to the last modification time of some file, I think they could get easily it in a shell script.

```
              + "  ts=timestamp  set last-modified timestamps:\n"
              + "    actual      use actual values in file-system\n"`
```

However, this doesn't need to to addressed in this PR (in case you agree).